simplesamlphp_auth.admin.inc

```diff
@@ -45,13 +45,18 @@ function simplesamlphp_auth_settings() {
     '#default_value' => $config->get('authsource'),
     '#description' => t('The name of the source to use from /var/simplesamlphp/config/authsources.php'),
   );
+  $form['simplesamlphp_auth_grp_setup']['simplesamlphp_auth_login_link_display_name'] = array(
```

Comment:
Try `$form['simplesamlphp_auth_grp_setup']['login_link_display_name'] = array(` here since we're simplifying the configuration setting's name for Backdrop.